bin/main.rb

```diff
@@ -1,59 +1,55 @@
-#!usr/bin/ruby
 # frozen_string_literal: true
 
-class Board
-  def initialize
-  @grid = [
-    ["-", "-", "-"],
-    ["-", "-", "-"],
-    ["-", "-", "-"]
-  ]
-
-  @top_row = "| | | |"
-  @mid_row = "| | | |"
-  @bot_row = "| | | |"
-
-  @instructions = ["|-|-|-|", "|-|-|-|", "|-|-|-|"]
-  end
-
-  def show
-    puts @top_row
-    puts @mid_row
-    puts @bot_row
-  end
-
-  def show_instructions
-    puts "Welcome to terminal tic tac toe!"
-    puts "The following are the move positions you are able to make:"
-    puts @instructions, ""
-  end
-
-  def check(a, b, player)
-    @grid[a][b] == player
-  end
-
-  def move(position, player)
-    puts "Pick a number to play"
+require_relative '../lib/game'
+class Playgame
```

Comment:
Convention for a class name is every new word should begin with a capital letter eg: `PlayGame`